tests/cmd/e2e/main.go

```diff
@@ -88,6 +93,7 @@ func main() {
 	}
 
 	restoreClusterInfo := &tests.TidbClusterInfo{
+		Name:             "test-backup",
```

Comment:
why the name as same as `clusterInfo` name?

Reply:
the name is pvc name, clusterinfo name is clustername in the test case